README.md

```diff
@@ -97,9 +97,13 @@ play around with Sapling as it currently stands, the best way is to clone the re
 from source (you'll need [Rust](https://www.rust-lang.org/learn/get-started) installed in order to do this):
 ```bash
 git clone https://github.com/kneasle/sapling.git
-cargo run
+cargo run 2> log
 ```
 
+Note that Sapling will not compile for Windows.  Windows support is absolutely intended, but Sapling currently uses
+[tuikit](https://github.com/lotabout/tuikit) as a terminal abstraction, which does not yet have Windows support.  PRs to
+Sapling or `tuikit` to add support for Windows would be very much appreciated.
+
```

Comment:
Ah yes these are changes that I committed directly to `master`.  If you do a simple rebase, they'll disappear:
```bash
git rebase master
git push origin --force-with-lease
```